chutney/ui/src/app/modules/scenarios/components/search-list/scenarios.component.ts

```diff
@@ -93,16 +94,11 @@ export class ScenariosComponent implements OnInit, OnDestroy {
             );
         });
 
-        this.getScenarios()
-            .then(r => {
-                this.scenarios = r || [];
-                this.applyDefaultState();
-                this.applySavedState();
-                this.applyUriState();
-                this.applyFilters();
-            })
-            .catch(err => console.log(err));
-
+        this.fetchAndUpdateScenario().pipe(map(scenarios => {
+            if (!this.noScenarioIsRunning(scenarios)) {
```

Comment:
sorry for my late comment, it will be easier to read with isScenarioRunning(...)

Reply:
I thing "isScenarioRunning" is not meaningful for a check over a list of scenarios, I renamed to "atLeastOneScenarioIsRunning"